typescript/syntax/Statements.sdf3

```diff
@@ -2,19 +2,28 @@ module Statements
 
 imports
 
+  Common
   Declarations
   Expressions
+  Functions
+ 
+context-free priorities
+
+  ReturnStatement.value > ReturnStatement.empty
```

Comment:
I tried to also fix the empty return, but was not able to. It still generates ambiguity because
```js
return function() {};
```
Can also be parsed as
```
return
function() {}
```
where the second is a declaration instead :sob: The priority didn't help here, so we can probably scratch that?